File: starting_filesets/C#/Yahtzee/YahtzeeTest.cs

using NUnit.Framework;

[TestFixture]
public class YahtzeeTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new Yahtzee().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/PHP/Yahtzee/YahtzeeTest.php
<?php

require_once 'PHPUnit/Framework.php';
require_once 'Yahtzee.php';

class YahtzeeTest extends PHPUnit_Framework_TestCase
{
    public function testAnswer()
    {
        $this->assertEquals(6 * 9, answer());
    }
}

?>
